### ISSUE TITLE
New-UDTooltip -type missing from 4.x

### ISSUE THREAD

**Issue by Omzig** [ironmansoftware/powershell-universal#2514]
### Version

4.0.3

### Severity

High

### Steps to Reproduce

```
New-UDTooltip -Content {
    New-UDIcon -Icon 'User'
} -TooltipContent {
    "User"
} -Type 'success'
```

An error occurred: A parameter cannot be found that matches parameter name 'Type'.
Endpoint: DashboardHeaderContent

https://docs.powershelluniversal.com/userinterfaces/components/data-display/tooltip

-type is missing in the live doc
-type exists in the api doc for the cmdlet https://github.com/ironmansoftware/universal-docs/blob/master/cmdlets/New-UDTooltip.txt

### Expected behavior

```console
-type works
```

### Actual behavior

```console
-type does not work
```

### Additional Environment data

IIS: Nested

**Comment by Omzig** [comment]
Need a 4.0.4 to assign it to

**Comment by adamdriscoll** [comment]
We moved to MUI's tooltip and it looks like this got removed somehow. 

Note that the v4 cmdlet docs are here and also do not have this parameter: https://github.com/ironmansoftware/universal-docs/blob/v4/cmdlets/New-UDTooltip.txt

**Comment by Omzig** [comment]
this did not make it to 4.0.5

**Comment by adamdriscoll** [comment]
We need to bump this again as we have some fixes we need to get out for 4.0.6 that are critical for some users. 

The reason this isn't a simple fix is that we switched to the MUI tooltip in v4 because it can be themed but MUI does not offer a built in Type property so we need to re-implement that manually .

**Comment by Omzig** [comment]
4.0.7 did not fix this

**Comment by adamdriscoll** [comment]
This failed to get merged from our master branch into 4.0. Will be included in the next version.
